lib/beaker/host_prebuilt_steps.rb

```diff
@@ -360,6 +360,10 @@ def copy_ssh_to_root host, opts
         else
           host.exec(Command.new('sudo su -c "cp -r .ssh /root/."'), {:pty => true})
         end
+
+        if host.exec(Command.new('sudo selinuxenabled'), :accept_all_exit_codes => true).exit_code == 0
```

Comment:
@trevor-vaughan this seems like something that would be great to have as a utility method of the host object. Would you create something along the lines of `selinux_enabled?` in the [unix](https://github.com/puppetlabs/beaker/tree/master/lib/beaker/host/unix) object?

If you just create an override of that method in the [windows](https://github.com/puppetlabs/beaker/tree/master/lib/beaker/host/windows) space (probably the [mac](https://github.com/puppetlabs/beaker/tree/master/lib/beaker/host/mac) space as well, I'm not familiar with selinux myself) that returns false since it's not applicable, that should cover the possibilities. Just to be complete, however, our host hierarchy looks like so:

- Unix
  - AIX
  - Cisco
  - EOS
  - FreeBSD
  - Mac
  - Windows
    - PSWindows

Don't worry if you're not familiar with some of these platforms. Having the main utility method in the Unix space should cover most of the possibilities needed here.